nature-logger/Controllers/PlantsDetailedViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  PlantsDetailedViewController.swift
+//  nature-logger
+//
+//  Created by Frederik Helth on 15/12/2020.
+//  Copyright © 2020 Syddansk Universitet. All rights reserved.
+//
+
+import UIKit
+
+class PlantsDetailedViewController: UIViewController {
+
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    
+    var hero:Plants.Result?
```

Comment:
Rename this to something else than "hero", maybe just "plant"? Looks too much like a copy-paste from an example :)